### PR TITLE
chore(trading): market setup fixtures

### PR DESCRIPTION
**Pull request by bwallacee**
intial idea on how to split out our market creation

### CODE REVIEW

File: marketFixtures/continuousMarket/continuousMarket.py
Comment (mattrussell36) [inline marketFixtures/continuousMarket/continuousMarket.py:1]: I believe the python convention is snake case.

File: tests/order_match/test_order_match.py [hunk 1]
```diff
@@ -168,9 +63,8 @@ def handle_response(route, request):
 
 
 @pytest.mark.usefixtures("auth")
-def test_limit_order_trade_open_order(vega, page):
-    # setup continuous trading market with one user buy trade
-    setup_continuous_market(vega, page)
+def test_limit_order_trade_open_order(setup_opening_auction_market, page: Page):
+    setup_opening_auction_market
```
Comment (mattrussell36) [inline tests/order_match/test_order_match.py:67]: Do you need to invoke the function?

**Review by mattrussell36 — CHANGES_REQUESTED** [review]
Dont see a problem with this. Though I'd probably try to lean into the pytest features and use fixtures for this

File: tests/order_match/test_order_match.py (hunk 1 above)
Comment (bwallacee) [reply tests/order_match/test_order_match.py:67]: Don't think we do, Will remove

File: marketFixtures/continuousMarket/continuousMarket.py
Comment (bwallacee) [reply marketFixtures/continuousMarket/continuousMarket.py:1]: Updated to snake case.

File: market_fixtures/opening_auction_market/opening_auction_market.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    submit_order(vega, MM_WALLET.name, market_id, "SIDE_SELL", 1, 105)
+    submit_order(vega, MM_WALLET2.name, market_id, "SIDE_BUY", 1, 95)
+
+    vega.forward("10s")
+    vega.wait_for_total_catchup()
+    
+
+    page.goto(f"http://localhost:{vega.console_port}/#/markets/{market_id}")
+
+    submit_order(vega, "Key 1", market_id, "SIDE_BUY", 1, 110)
```
Comment (daro-maj) [inline market_fixtures/opening_auction_market/opening_auction_market.py:100]: Do we need this ? Trade is only needed to switch to continuous mode

**Comment by bwallacee** [comment]
I think for now this way is "ok" but we may want to look into creating a new conftest.py and having the setups moved to there.

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**